test/functional/page_objects/settings_page.js

```diff
@@ -45,7 +45,7 @@ export function SettingsPageProvider({ getService, getPageObjects }) {
 
     async clickKibanaSettings() {
       await this.clickLinkText('Advanced Settings');
-      await PageObjects.header.waitUntilLoadingHasFinished();
+      await testSubjects.exists('managementSettingsTitle');
```

Comment:
I think there is some confusion about what this test does.  It returns true or false, it doesn't assert that that does indeed exist. I think the default time is like 1 second, so this is the equivalent of adding a 1 second sleep. :)  If you want to assert that it does exist, use find instead, as that will retry for something like 60 seconds, and fail if it is never found.

The function should probably be renamed `getExists` instead of `exists` to eliminate that confusion.

Reply:
Good catch.  It does return true or false.  And it has a 1 second timeout, but I don't think that means it's like a 1 second sleep.  If it does find it it will return as quickly as it can.  If it doesn't find it within 1 second it will return false.

Reply:
Well, in the latter case, if it doesn't find it, it's like adding a 1 second sleep.  In the former case, it's like adding a variable sleep, up to 1 seconds, possibly quicker.  I'm wondering if that's why the tests appear more stable.